src/main/grammars/WebAssemblyParser.bnf

```diff
@@ -15,6 +15,72 @@
 
 webAssemblyFile ::= item_*
 
-private item_ ::= (property|COMMENT|CRLF)
+private item_ ::= comment | module | modulefield+
 
-property ::= (KEY? SEPARATOR VALUE?) | KEY
\ No newline at end of file
+comment ::= LINE_COMMENT | blockcomment
+blockcomment ::= COMMENT_LEFT_BR (COMMENT_CHAR | blockcomment)* COMMENT_RIGHT_BR
```

Comment:
comments must be processed at the lexing stage (see my comment above) otherwise parser cannot handle it properly.